13-TeV-examples/uproot_python/HyyAnalysisNew.ipynb

```diff
@@ -164,526 +164,551 @@
-    "tuple_path = \"https://atlas-opendata.web.cern.ch/atlas-opendata/13TeV/GamGam/Data/\" # web address"
+    "path = \"https://atlas-opendata.web.cern.ch/atlas-opendata/13TeV/GamGam/Data/\" # web address"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<a id='samples'></a>"
+    "The data is organized by the collection periods throughout the year. In this notebook, we will use the 2015 data from periods D, E, F, G, H, and J, as well as the 2016 data from periods A, B, C, D, E, F, G, K, and L."
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": 4,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "## Samples\n",
-    "\n",
-    "Samples to process"
+    "samples_list = ['data15_periodD','data15_periodE','data15_periodF','data15_periodG','data15_periodH','data15_periodJ','data16_periodA','data16_periodB','data16_periodC','data16_periodD','data16_periodE','data16_periodF','data16_periodG','data16_periodK','data16_periodL']"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "samples_list = ['data15_periodD','data15_periodE','data15_periodF','data15_periodG','data15_periodH','data15_periodJ','data16_periodA','data16_periodB','data16_periodC','data16_periodD','data16_periodE','data16_periodF','data16_periodG','data16_periodK','data16_periodL']"
+    "Let's try accessing `data15_periodG` in the CERN database URL as an example."
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": 5,
    "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "data15_periodG\n"
+     ]
+    }
+   ],
    "source": [
-    "[Back to contents](#contents)"
+    "print(samples_list[3])\n",
+    "\n",
+    "# This is now appended to our file path to retrieve the data15_periodG.root file\n",
+    "data_15G_path = path + samples_list[3] + \".root\""
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Define function to get data from files\n",
+    "Next, let's open the `data15_periodG` file to examine its contents. The file, known as a `tree`, contains multiple entries, each representing an event. For each event, a dictionary stores all relevant information with keys, such as the event number (`eventNumber`), the photon transverse momentum (`photon_pt`), and more.\n",
+    "\n",
+    "Details on the variables in the dictionary can be viewed [here](https://cds.cern.ch/record/2707171/files/ANA-OTRC-2019-01-PUB-updated.pdf) in Appendix A.\n",
     "\n",
-    "The datasets used in this notebook have already been filtered to include at least 2 photons per event, so that processing is quicker."
+    "More information on trees can be viewed [here](https://masonproffitt.github.io/uproot-tutorial/03-trees/index.html)."
```

Comment:
Is this still the right link, to Mason's github area?

Reply:
OH I see, this is for documentation of what a tree is in uproot? If so, maybe we want
https://uproot.readthedocs.io/en/latest/uproot.behaviors.TTree.TTree.html
and
https://hsf-training.github.io/hsf-training-uproot-webpage/03-trees/index.html
? The latter seems a bit more 'stable' than Mason's area, I think?